src/dmd/root/array.d

```diff
@@ -313,3 +313,15 @@ unittest
     array[2] = 910;
     assert([123, 421, 910, 123, 1, 2, 8, 20, 4, 3] == array.asDArray);
 }
+
```

Comment:
why this ?

Reply:
Allows you to compare to pointers to array and handles the `null` case for you as well

Reply:
ok, changed it to `opEquals`.

Reply:
Why not `left.asDArray == right.asDArray`?

Reply:
Would have to be `(*left).asDArray` and `(*right).asDArray`, and this segfaults when left or right is null.